### PR TITLE
Add sample for NServiceBus.Extensions.Logging v2

### PR DESCRIPTION
**Pull request by timbussmann**
Also changes the log level for the handler message to warn since we're using a colored logger which isn't clear at all as long as we're only logging info/debug messages.

### CODE REVIEW

File: samples/logging/extensions-logging/Extensions.Logging_1/Sample/MyHandler.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class MyHandler :
 
     public Task Handle(MyMessage message, IMessageHandlerContext context)
     {
-        log.Info("Hello from MyHandler");
+        log.Warn("Hello from MyHandler");
```
Comment (HEskandari) [inline samples/logging/extensions-logging/Extensions.Logging_1/Sample/MyHandler.cs:12]: should this be a WARN?

**Review by mikeminutillo — APPROVED** [review]
I'm not sure about switching to WARN for visual clarity but not enough to block this

File: samples/logging/extensions-logging/Extensions.Logging_1/Sample/MyHandler.cs (hunk 1 above)
Comment (timbussmann) [reply samples/logging/extensions-logging/Extensions.Logging_1/Sample/MyHandler.cs:12]: see the comment on the isse description

**Comment by timbussmann** [comment]
for more context:

With `Info`:
image


with `Warn`:
image

**Comment by timbussmann** [comment]
discussed the log level. Merging this for now since it can always be changed easily.

**Pull request merged**